etc/init.d/zfs-zed.in

```diff
@@ -45,8 +39,7 @@ ZED_PIDFILE="@runstatedir@/$ZED_NAME.pid"
 
 do_depend()
 {
-	before zfs-import
-	after sysfs
+	after sysfs zfs-mount
```

Comment:
Thinking about this some more, I realize it makes more sense in the general case to have `zfs-zed` depend on `localmount` instead, which in turn depends on `zfs-mount` when `/` is ZFS, and supports the case when `/` is not ZFS and `/var` is a separate ext4 filesystem, as another example.  Do you agree?